--- git status ---
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   scripts/reinforcement_learning/rsl_rl/play.py
	modified:   source/isaaclab_tasks/isaaclab_tasks/direct/anymal_c/anymal_c_env_cfg.py

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	source/isaaclab_tasks/isaaclab_tasks/direct/anymal_c/terrain_generator_cfg.py
	source/isaaclab_tasks/isaaclab_tasks/direct/anymal_c_custom/
	source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/anymal_c_custom/

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/scripts/reinforcement_learning/rsl_rl/play.py b/scripts/reinforcement_learning/rsl_rl/play.py
index e3c0121e3..07e88ec25 100644
--- a/scripts/reinforcement_learning/rsl_rl/play.py
+++ b/scripts/reinforcement_learning/rsl_rl/play.py
@@ -148,6 +148,16 @@ def main():
             actions = policy(obs)
             # env stepping
             obs, _, _, _ = env.step(actions)
+
+            v_x = obs["v_x"].item() if "v_x" in obs else None
+            omega_z = obs["omega_z"].item() if "omega_z" in obs else None
+            avg_tracking_error_x = obs["avg_tracking_error_x"].item() if "avg_tracking_error_x" in obs else None
+            avg_tracking_error_omega_z = obs["avg_tracking_error_omega_z"].item() if "avg_tracking_error_omega_z" in obs else None
+
+            print(f'    "v_x": {v_x},')
+            print(f'    "omega_z": {omega_z},')
+            print(f'    "avg_tracking_error_x": {avg_tracking_error_x},')
+            print(f'    "avg_tracking_error_omega_z": {avg_tracking_error_omega_z}')
         if args_cli.video:
             timestep += 1
             # Exit the play loop after recording one video
@@ -167,4 +177,4 @@ if __name__ == "__main__":
     # run the main function
     main()
     # close sim app
-    simulation_app.close()
+    simulation_app.close()
\ No newline at end of file
diff --git a/source/isaaclab_tasks/isaaclab_tasks/direct/anymal_c/anymal_c_env_cfg.py b/source/isaaclab_tasks/isaaclab_tasks/direct/anymal_c/anymal_c_env_cfg.py
index 24bed6353..e6371a9e6 100644
--- a/source/isaaclab_tasks/isaaclab_tasks/direct/anymal_c/anymal_c_env_cfg.py
+++ b/source/isaaclab_tasks/isaaclab_tasks/direct/anymal_c/anymal_c_env_cfg.py
@@ -20,6 +20,7 @@ from isaaclab.utils import configclass
 ##
 from isaaclab_assets.robots.anymal import ANYMAL_C_CFG  # isort: skip
 from isaaclab.terrains.config.rough import ROUGH_TERRAINS_CFG  # isort: skip
+from .terrain_generator_cfg import get_multiple_terrains_cfg, get_unique_terrain_cfg
 
 
 @configclass
@@ -118,7 +119,7 @@ class AnymalCRoughEnvCfg(AnymalCFlatEnvCfg):
     terrain = TerrainImporterCfg(
         prim_path="/World/ground",
         terrain_type="generator",
-        terrain_generator=ROUGH_TERRAINS_CFG,
+        terrain_generator=get_multiple_terrains_cfg(num_rows=2, num_cols=5),
         max_init_terrain_level=9,
         collision_group=-1,
         physics_material=sim_utils.RigidBodyMaterialCfg(